src/net.rs

```diff
@@ -6,24 +6,28 @@ use std::{
     pin::Pin,
     sync::{atomic::AtomicUsize, Arc},
     task::{Context, Poll},
-    time::Instant,
 };
 
 use bytes::BytesMut;
 use futures_concurrency::stream::{stream_group, StreamGroup};
-use futures_lite::{future::Boxed as BoxedFuture, stream::Stream, StreamExt};
-use futures_util::TryFutureExt;
+use futures_util::FutureExt as _;
```

Comment:
is there nothing for this in `n0-future` yet?

Reply:
The module uses [`futures_util::FutureExt::map`](https://docs.rs/futures-util/latest/futures_util/future/trait.FutureExt.html#method.map)
[`n0_future` reexports `futures_lite::FutureExt`](https://docs.rs/n0-future/latest/n0_future/trait.FutureExt.html) only which does not have that.

Reply:
Added this to https://github.com/n0-computer/n0-future/issues/1